packages/rich-text/src/to-tree.js

```diff
@@ -305,6 +305,8 @@ export function toTree( {
 						// selection. The placeholder is also not editable after
 						// all.
 						contenteditable: 'false',
+						style:
+							'pointer-events:none;user-select:none;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;',
```

Comment:
Never ran into [user-select](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select) before, TIL.

Reply:
Yeah, everything selection related is a bit obscure